src/Backoffice/Airline/App/Transformers/AirlineTransformer.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Airline\App\Transformers;
+
+use Flugg\Responder\Transformers\Transformer;
+use Lightit\Backoffice\Airline\Domain\Models\Airline;
+use Lightit\Backoffice\City\App\Transformers\CityTransformer;
+
+class AirlineTransformer extends Transformer
+{
+    public function transform(Airline $airline): array
+    {
+        $citiesTransformed = $airline->cities->map(function ($city) {
+            return (new CityTransformer())->transform($city);
+        });
```

Comment:
You can use arrow funcs if you're only returning inside it

```suggestion
        $citiesTransformed = $airline->cities->map(fn ($city) => (new CityTransformer())->transform($city));
```